arviz/plots/densityplot.py

```diff
@@ -64,7 +65,7 @@ def plot_density(
         Defaults to 0.94.
     point_estimate : Optional[str]
         Plot point estimate per variable. Values should be 'mean', 'median', 'mode' or None.
-        Defaults to 'auto' i.e. it falls back to default set in rcParams.
+        Defaults to 'auto' i.e. it falls back to default set in `rcParams`.
```

Comment:
single backticks are used for italic text. Use double backticks for highlighting code keywords. ` ``rcParams`` `